### DISCUSSION TITLE
Charges

### DISCUSSION THREAD

**Discussion by choutkaj** [grimme-lab/xtb#902] [General]
Hi! 

I am new to xtb, just want to clarify. In the documentation, it is stated that the total charge of the system should be specified either in .CHRG file, or within input by command --chrg. However, when I run xtb from pdb or sdf input, the charge of the system is read automatically without the need for any of that. Is it OK to rely on this and run xtb without specifying charge, or are there any problems with it?

Thank you for your answer :)

**Comment by Albkat** [comment]
Hello,
It's a good practice to double-check the charges specified by any molecular data formats during calculation. The reliability of those charges depends on their source, whether from calculations, experiments, or both.

**Reply by choutkaj** [reply]
OK so if I understand correctly, the best way to use xtb is to specify the total charge in the input, but if I don't do that the program will attempt to calculate the total charge automatically if it has somewhere to read it from (for example pdb or sdf file which contain formal charges). Is that right?

**Reply by Albkat** [reply]
Correct, and if nothing is found, it will then assign the default value of **0**.
Additionally, there exists a hierarchy (arranged from highest to lowest priority) for charge initialization:
`--chrg` CLI argument > **xcontrol** instruction > **.CHRG** > from molecular data file

**Reply by choutkaj** [reply]
Great! Thanks for the info.
